TAs/optee_ta/AuthVars/Makefile

```
@@ -1,7 +1,8 @@
 BINARY=2d57c0f7-bddf-48ea-832f-d84a1a219301
 
 O ?= ../out/AuthVars
-WOLF_ROOT := ../../../../external/wolfssl/
+WOLF_ROOT := ../../../external/wolfssl/
+OSSL_ROOT := ../../../external/ossl_lib
```

Should be renamed to 'ossl' for consistency
```suggestion
OSSL_ROOT := ../../../external/ossl
```

---

ossl_lib contains all the static compilation and extern references to the stdlib. It then itself contains the submodule /openssl. I can certainly change it though.

---

Please do.  I think consistent naming is beneficial here especially for those unfamiliar with the gory details (even at the cost of absolute exactitude).